packages/mobile/src/fiatExchanges/MoonPay.tsx

```diff
@@ -34,11 +32,9 @@ type Props = RouteProps
 
 function FiatExchangeWeb({ route }: Props) {
   const [uri, setUri] = React.useState('')
-  const { amount } = route.params
+  const { localAmount, currencyCode } = route.params
   const account = useSelector(currentAccountSelector)
-  const localCurrencyCode = useSelector(getLocalCurrencyCode)
```

Comment:
Moved currency conversion logic to FiatExchangeOptions as we may want to display the different providers' different fiat, CELO and cUSD rates down the line